src/components/testimonials-items/testimonial-item.jsx

```diff
@@ -1,13 +1,13 @@
-import React, { useState } from 'react';
+import React from 'react';
 
-const TestimonialsItem = ({ img, author, description }) => {
+const TestimonialsItem = ({ item }) => {
```

Comment:
if the props are the same, and are just wrapped by this object, you may as well just prop spread? at least, not sure why this change is needed

prop spread means where this is called, do `<TestimonialsItem {...item} />` and revert these changes.